editor/editor_resource_picker.cpp

```diff
@@ -71,10 +72,15 @@ void EditorResourcePicker::_update_resource() {
 			} else if (edited_resource->get_path().is_resource_file()) {
 				assign_button->set_text(edited_resource->get_path().get_file());
 			} else {
-				assign_button->set_text(edited_resource->get_class());
+				assign_button->set_text(class_name);
 			}
 			assign_button->set_tooltip_text(resource_path + TTR("Type:") + " " + edited_resource->get_class());
 
+			if (edited_resource->get_path().is_resource_file()) {
+				resource_path = edited_resource->get_path() + "\n";
+			}
+			assign_button->set_tooltip_text(resource_path + TTR("Type:") + " " + class_name);
+
```

Comment:
A rebase error? The assignment already happens way above on lines 53-55. I guess you want to replace `edited_resource->get_class()` with `class_name` on line 77?